app/controllers/consultees_controller.rb

```diff
@@ -3,12 +3,29 @@
 class ConsulteesController < AuthenticationController
   before_action :set_planning_application
 
+  def edit
+    @consultee = planning_application.consultees.find(Integer(params[:id]))
```

Comment:
nitpick: since this is the same as finding the consultee in the update action we could put this into a private method and have a before_action ?